### DISCUSSION TITLE
Is it possible to insert processing before and after creating or updating?

### DISCUSSION THREAD

**Discussion by narita-takeru** [simolus3/drift#1863] [Q&A]
I like drift very much.
Ask a trivial question.

For example, ruby's ActiveRecord has `before_save / before_update` and` after_save / after_update`.
Does Drift offer the same functionality as those?

**Comment by simolus3** [comment]
Not really. You can set a [computable function](https://drift.simonbinder.eu/api/drift/buildcolumn/clientdefault) as a default value for columns which would then be invoked then the column is not set on inserts. But that's not really a `before_save` and it doesn't work for updates at all. There's no callback that would be invoked after an operation either.

Do you have a specific use-case in mind that would require these callbacks?
